### DISCUSSION TITLE
Create Issued Document - Error 409 Cannot perform the operation

### DISCUSSION THREAD

**Discussion by pws2016** [fattureincloud/api#21] [Q&A] [answered]
Hello i try to create new invoice and i set data as in api documentation: https://developers.fattureincloud.it/docs/guides/invoice-creation#-create-the-document

but is returned error:
Exception when calling IssuedDocumentsApi->createIssuedDocument: [409] Client error: `POST https://api-v2.fattureincloud.it/c/884109/issued_documents` resulted in a `409 Conflict` response: {"error":{"message":"Cannot perform the operation."}}

what's wrong?!

**Comment by emajo** [comment]
Hi @pws2016, I'm guessing you are on the 'prova forfettari' plan and you are trying to create an invoice with a vat different from 0% which is not allowed.

Am I right?

**Reply by pws2016** [reply]
i set vat id=0 (22%) the default vat

**Reply by emajo** [reply]
yes, and your plan is a 'forfettari' plan, which does not allow the creation of invoices with vat, you will have to change plan if you want to create invoices with 22% of vat

**Reply by pws2016** [reply]
so what's the id vat of 0% ?

**Reply by emajo** [reply]
you can make a [List Vat Types](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/info/vat_types) request to see all the vat type related to your company

**Comment by valmoz** [comment, marked as answer]
> you can make a [List Vat Types](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/info/vat_types) request to see all the vat types related to your company

Marking @emajo message as the correct answer
